borg/_hashindex.c

```diff
@@ -77,6 +79,7 @@ static int hash_sizes[] = {
 
 #define HASH_MIN_LOAD .25
 #define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_EFF_LOAD .85
```

Comment:
Hm, I didn't mean to say that 0.93 would be more correct, just that 0.85 seems to be the right ballpark. If in doubt, use the more conservative figure (0.85)?